packages/odata-service-writer/test/index.test.ts

```diff
@@ -49,12 +49,13 @@ describe('ODataService templates', () => {
      * @param name testDir name
      * @returns testDir Path
      */
-    async function createTestDir(name: string): Promise<string> {
+    async function createTestDir(name: string, serviceType?: ServiceType): Promise<string> {
         const testDir = join(outputDir, name);
         const ui5Yaml = (await UI5Config.newInstance('')).addFioriToolsProxydMiddleware({ ui5: {} }).toString();
         fs.write(join(testDir, 'ui5.yaml'), ui5Yaml);
         fs.write(join(testDir, 'ui5-local.yaml'), '');
-        fs.writeJSON(join(testDir, 'package.json'), { ui5: { dependencies: [] } });
+        const packageJson = serviceType === ServiceType.CDS ? {} : { ui5: { dependencies: [] } };
+        fs.writeJSON(join(testDir, 'package.json'), packageJson);
```

Comment:
```suggestion
```
I don't think we need to do this update at all. `ui5: { dependencies: [] }` was needed for @ui5/cli V1 and V2. It is no longer needed in V3 which we currently write to the project.

Reply:
I have removed this dependency.